packages/web-app-mail/src/components/MailAttachmentItem.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <div class="mail-attachment-item flex justify-between items-center">
```

Comment:
Hmm I often see classes on elements like here e.g. `mail-attachment-item`. Do you need them? One of the advantages with Tailwind is that you don't need custom css class names anymore since they just bloat the DOM.